### DISCUSSION TITLE
Total usage for each User

### DISCUSSION THREAD

**Discussion by BlackWand1** [minio/minio#17528] [Q&A] [answered]
Hi!

Is it possible to somehow see the total usage, for all buckets, related to a specific user?

I cant find no such feature in gui or when "googling" for it.
In the bucket browser I can get a nice overview, but it only lists the bucket name, not the user.

Thanks.

**Comment by klauspost** [comment, marked as answer]
Objects are not attached to users, so there is no such functionality.

If users have separate buckets, you can use the bucket stats. I believe the latest version exports these as a metrics.

**Reply by BlackWand1** [reply]
OK. So if I somehow want to keep track of buckets usage and each "owner" so that I can invoice them. How would I go about doing that?

**Reply by klauspost** [reply]
As I wrote bucket stats are exported as metrics. These are lazy stats.

Alternatively set up audit logging, record operations and bill on derived usage.

Since you are building a commercial setup I suggest you reach out to `hello@min.io` and we can go into more details.

**Reply by BlackWand1** [reply]
I will reach out using that email. Thank you.
